scripts/setup

```diff
@@ -292,7 +292,9 @@ eval ${PREFIX} invenio utils reindex -t cipo --yes-i-know --no-info
 eval ${PREFIX} invenio utils runindex --raise-on-error
 
 info_msg "- Users: ${DATA_PATH}/users.json"
-eval ${PREFIX} invenio fixtures import_users ${DATA_PATH}/users.json -v
+eval ${PREFIX} invenio fixtures import_users ${DATA_PATH}/users.json -v --append
```

Comment:
We have no test if a user exist already any more if we index all users at the end. See `get_pid_by_email`!